### DISCUSSION TITLE
SOAP-ERROR: Parsing WSDL: Couldn't load from <URL>

### DISCUSSION THREAD

**Discussion by liana13** [CodeDredd/laravel-soap#305] [Q&A]
Can you help me please?
I have this code 

`return Soap::baseWsdl(config('soap.clients.1c_soap.base_wsdl'))
            ->withBasicAuth(config('soap.clients.1c_soap.with_basic_auth.username'),config('soap.clients.1c_soap.with_basic_auth.password'))
            ->call('GetItemsForSale',[
                'MaxCount' => 10,
                'AutoConfirm' => true,
            ]);`

and get an error

SOAP-ERROR: Parsing WSDL: Couldn't load from <URL> : failed to load external entity <URL>

**Comment by sc85** [comment]
Have a look if `config('soap.clients.1c_soap.base_wsdl')` contains an URL and if the WSDL at this URL is valid.

**Reply by liana13** [reply]
Thanks for your answer. The config('soap.clients.1c_soap.base_wsdl') contains an URL and if the WSDL at this URL is valid. I think the problem is with Auth. config('soap.clients.1c_soap.with_basic_auth.username'),config('soap.clients.1c_soap.with_basic_auth.password') contains the right username and password. I just checked in a browser the url and basic authentication.
Is there any misconfiguration?

`'clients' => [
        '1c_soap' => [
            'base_wsdl' => env('1C_URL'),
            'with_wsa' => false,
            'with_basic_auth' => [
                'username' => env('1C_USERNAME'),
                'password' => env('1C_PASSWORD'),
            ],
            'with_wsse' => [
                'user_token_name' => 'username',
                'user_token_password' => 'password',
                'private_key_file' => 'path/to/privatekey.pem',
                'public_key_file' => 'path/to/publickey.pyb',
                'server_certificate_file' => 'path/to/client-cert.pem',
                'server_certificate_has_subject_key_identifier' => false,
                'user_token_digest' => false,
                'digital_sign_method' => XMLSecurityKey::RSA_SHA1,
                'timestamp' => 3600,
                'sign_all_headers' => false,
            ],
        ],
    ],`
